handbook/company/product-groups.md

```diff
@@ -536,6 +536,75 @@ PRs should remain in draft until they are ready to be reviewed for final approva
 
+> This script is intended to run on macOS. The usage of `date` is incompatible on linux, you will need to switch `date -j -f "%Y-%m-%dT%H:%M:%SZ" "$created_at" "+%s"` to `date -d "$created_at" "+%s"`
+
+```bash
+#!/bin/bash
+
+# Set default values for ORG and REPO if not defined externally
+ORG="${ORG:-fleetdm}"
+REPO="${REPO:-fleetdm/fleet}"
+
+# Ensure GITHUB_TOKEN is defined
+if [ -z "$GITHUB_TOKEN" ]; then
+    echo "Error: GITHUB_TOKEN is not defined. Go to https://github.com/settings/tokens and generate a new token to continue."
+    exit 1
+fi
+
+# Function to fetch members and handle pagination
+fetch_members() {
+    local url=$1
+    local logins=()
+    while [ -n "$url" ]; do
+        response=$(curl -s -H "Authorization: token $GITHUB_TOKEN" \
+                          -H "Accept: application/vnd.github.v3+json" \
+                          "$url")
+        logins+=($(echo "$response" | jq -r '.[].login'))
+
+        # Extract the next URL from the Link header
+        url=$(curl -s -I -H "Authorization: token $GITHUB_TOKEN" \
+                      -H "Accept: application/vnd.github.v3+json" \
+                      "$url" | awk -F'[<>]' '/<[^>]+>; rel="next"/ {print $2}')
+    done
+    echo "${logins[@]}"
+}
+
+members=($(fetch_members "https://api.github.com/orgs/$ORG/members?role=all"))
+echo "Number of fleet members excluding from query: ${#members[@]}"
+if [ ${#members[@]} -lt 30 ]; then
+     echo "Abnormally low number of fleet members detected.
+Please check that your access token has permissions to query \"private\" members of the fleet organization.
+This requires the \"read:org\" scope."
+fi
+
+# query open prs from authors that are not a member of fleet org
+query="is:pr repo:$REPO is:open"
+for member in "${members[@]}"; do
+     query="$query -author:$member"
+done
+# add dependabot manually
+query="$query -author:dependabot[bot]"
+
+encoded_query=$(jq -rn --arg q "$query" '$q|@uri')
+search_url="https://api.github.com/search/issues?q=$encoded_query"
+echo "Search URL: https://github.com/search?q=$encoded_query"
+response=$(curl -s -H "Authorization: token $GITHUB_TOKEN" \
+                 -H "Accept: application/vnd.github.v3+json" \
+                 "$search_url")
+
+# Output the count of items in the response
+count=$(echo "$response" | jq '.total_count')
+echo "Number of results: $count"
+
+# Output the response in a table format
+echo "$response" | jq -r '.items[] | "\(.html_url)\t\(.user.login)\t\(.created_at)\t\(.title)"' | while IFS=$'\t' read -r url user created_at title; do
+    created_at_epoch=$(date -j -f "%Y-%m-%dT%H:%M:%SZ" "$created_at" "+%s")
+    current_epoch=$(date "+%s")
+    days_ago=$(( (current_epoch - created_at_epoch) / 86400 ))
+    printf "%s#@#%s#@#%s\n" "$url" "$user created $days_ago days ago" "$title"
+done | column -s '#@#' -t
+```
```

Comment:
I would move this whole script into `tools/oncall` and then reference it here.

Feel free to remove the ones already in tools/oncall/ if this script replaces them (they haven't been used/udpated for a while).